homeassistant/components/synology_dsm/binary_sensor.py

```diff
@@ -76,8 +71,3 @@ def is_on(self) -> bool:
         if attr is None:
```

Comment:
Maybe `getattr(self._api.storage, self.entity_type)(self._device_id) != "normal"`, I need to confirm that.

Storage (binary) sensors deos not return a bool but a status that we have to check.

Otherwise the state will always be `on` !

Reply:
See 1 line up